### DISCUSSION TITLE
Does REM stop dosing if tanks are too low?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#757] [Q&A]
If so, what is the minimum tank level for dosing to occur?

Can it be adjusted?

**Comment by rstrouse** [comment]
0%.  You can set an alarm under the alarms tab.  If you have calibrated your pump you will see that it is pretty darn accurate.

**Reply by davthomaspilot** [reply]
I calibrated the Intellichem acid pump--it was very close to 150 ml/minute.

I didn't bother with the chlorine pump--I figured it would be the same.  But, REM is reporting empty tank way too soon.

I know when the pump tubes are worn or the rollers in the pump head have their ID enlarged due to long time use, the pump won't inject anything at all when connected to the injector.  They will pump ok when just pumping into a measuring cup (no back pressure).

But, I've replaced the roller assembly and have a new pump tube.  I'll do the calibration on the chlorine some time soon.  

I may have to replace the pumps anyway.  I bought 15 gallon tanks from Stenner.   The Stenner tech rep says I can't use the Intellichem pumps because they are integrated into the tank lid assembly.  

Based on the pics and the pump tube replacements I've used, I think the Intellichem pumps are the Econ series except for integration into the tank lid.  

I'm hoping I can 3d print an enclosure that will mount on the 15 gallon tank and is compatible with the "guts" of the Intellichem pumps.  If not, I'll buy replacement Stenner pumps, Econ series, 24V.

**Comment by davthomaspilot** [comment]
I would like the dosing to be inhibited before tank is 0%. 

 If the suction tube picks up any air, often the pump won't inject anything at all.  So, I need to get more chlorine in the tank BEFORE the tank is empty.

I'm getting that behaviour anyway, since the chlorine pump is apparently not pumping the 150 ml/minute that I've configured in REM.

So, it's not a big deal, since I can configure REM ml/min higher than it actually is and get what I want.

But, don't others have this same issue?  Can't run tanks to empty without getting air bubbles in line?

**Reply by johnny2678** [reply]
I have the Pentair tank, so maybe it's different with Stenner but when my tank's empty it just pushes air through the line.  Then resumes pushing acid when I fill the tank.

**Reply by davthomaspilot** [reply]
Yeah, mine doesn't (I have the Pentair too--haven't moved to the Stenner tank yet)

Sometimes the acid pump will continue after pumping air, but the chlorine never does.  I have to disconnect the line and bleed the air out.

Maybe the length of the tubing is the difference.

Anyway, rather than giving a different pump rate to REM, I can just change the tank level--sort of like "useable fuel" in an airplane.  You can't use it all, since the fuel pick-up can never get 100% out of the tank.

So, I'll set the tank level to something like 14 instead of 15 for the new 15 gallon tank and should never run pumps dry again.

(The Pentair controller doesn't stop dosing even when it thinks tanks are empty--Hooray REM!

**Reply by truvec** [reply]
I understand that you have a work around but there should be no issue for this pump to re prime.  Best guess is the rollers are not fully sealing the tube. This would cause it to not prime. This would also allow the chlorine to “leak” back as it’s pumping. Causing rem to think it’s pumping more than it is.

**Reply by rstrouse** [reply]
You guys are really overthinking this.  My Pentair tank leaves maybe 10mL in the bottom of the tank when it goes empty. If I get a warning I add a gallon or two.  Unless I want it to go empty it never does.  Also in 1/4 inch tubing the quantity is remarkably small.  That being said IntelliChem is really braindead about how it goes about processing alarms.  You only get a 1/6 tank setting so the last portion of the tank will leave you pumping long after the tank is empty.  To add to the goober a tank level of -1 is empty not 0.

The design of a peristaltic pump means that it should never need to be primed.  If the suction tube is in liquid it will pull that liquid.  That is unless you do not have a proper check valve on the injector or the pump tube needs replaced.  You need to verify those two things on your pumps if that is not the case.  In some cases you can run without a check valve but the tube needs to be in good shape or it will expand as the first roller goes over it.  This leaves a lack of volume to push the air out on the last roller.

**Comment by davthomaspilot** [comment]
Well,  I can tell you with new head, new check valve,and new pump tube it won't inject chlorine  if there is air in the line.

It is not the pump doesn't "prime" or doesn't get suction.  + The pump just compresses  the air in the output  line instead of pushing the acid through the injection valve.

Whether it's a problem or not depends on how much air in the line (how long the line is from the pump to the injection valve) and how much pressure is in the line.

In turn, the pressure depends on what RPM you are running the pump.

It's always been this way since Intellichem was first installed.

Maybe my dosing tube length from the pump to the injection valve is longer than most, or maybe I'm running my pump at a higher RPM.
But, I learned years ago to avoid the hassle by not letting the pump suck air.

Certainly it happens when the tube is worn or when the head rollers are worn.  I just went through all that last week and let the pump run dry.  But, I had to bleed the air out, even with everything new.


    On Thursday, May 11, 2023 at 02:20:28 PM EDT, rstrouse ***@***.***> wrote:  
 
 


You guys are really overthinking this. My Pentair tank leaves maybe 10mL in the bottom of the tank when it goes empty. If I get a warning I add a gallon or two. Unless I want it to go empty it never does. Also in 1/4 inch tubing the quantity is remarkably small. That being said IntelliChem is really braindead about how it goes about processing alarms. You only get a 1/6 tank setting so the last portion of the tank will leave you pumping long after the tank is empty. To add to the goober a tank level of -1 is empty not 0.

The design of a peristaltic pump means that it should never need to be primed. If the suction tube is in liquid it will pull that liquid. That is unless you do not have a proper check valve on the injector or the pump tube needs replaced. You need to verify those two things on your pumps if that is not the case. In some cases you can run without a check valve but the tube needs to be in good shape or it will expand as the first roller goes over it. This leaves a lack of volume to push the air out on the last roller.

**Reply by truvec** [reply]
Since everything else is new. 
Have you checked the suction tubing? (from the pump to the tank). 

On the supply side the pump has no issue injecting up to 80psi. So it’s probably not due that side of the pump. Have you run the pump with that side disconnected when trying to prime? It doesn’t need a check valve to work. On my chlorine tanks I’ll disconnect the line and put my finger on the port to make sure it’s sucking or pushing(depending on which side of the roller). It will be evident quite quickly if the tube is good. I’ll note that my tank has diluted chlorine so in this case it would make sense to use a glove and maybe skip this test for the acid tank. Also wear eye protection. Etc. 

Sorry to beat a dead horse but I wouldn’t be satisfied with a stenner pump that wouldn’t self-prime. They just work. And when they don’t there’s just a few places to check.

**Reply by davthomaspilot** [reply]
I'm happy to beat a dead horse--

It's not a "priming" issue in the sense that something needs to get moist and swell before suction occurs.  

After the tank runs to empty, I disconnect the tubing at the check valve.  It will pump spurts of chlorine with periods of nothing coming out but air.

I do that for 20 or 30 seconds until there are no air bubbles coming out.  Then I reconnect to the check valve and all is good (until I run the tank to empty again.

The suction side is a piece of tubing inserted directly into a 5 gallon container of 12%  sodium hypochlorite.  I stopped bothering with pouring the 5 gallons of hypochlorite into the Intellichem container and just put the tube in another 5 gallon container.

I suppose it's possible that the tube fittings at the pump aren't air tight.  But, this problem is not unique to me.  I only knew to bleed the air out of the line by googling the issue and finding that as a solution.  And, it has always done that.

Sometimes, the acid pump will work after it pumps air, but usually not.  The chlorine pump will NEVER pump into the injector until I bleed the line.

If anyone is still interested and willing to "weigh in":

It's getting close to empty now.  I'll let it run dry and make a YouTube of what the output when I disconnect the tubing from the injects and post a link here.

If I could figure out a way to get an air tight connection to my compressor, I could also test for leaks that way.   In the past, I've just blown into the tubes with my finger over the other end.  But, that wouldn't be near the psi required to actually inject chlorine.

**Reply by davthomaspilot** [reply]
Thinking more about how to demo this...

I'll just remove the tube from the 5 gallon container and let the pump continue dosing until the tubing is full of air. 

 Then, I'll put the tube into a cup of chlorine and show that no chlorine get sucked from the cup.  

Then, I will disconnect from the injector and let the pump runs until no air comes out.  Then, I will put the suction side back into the cup of chlorine and show it now works fine

**Reply by rstrouse** [reply]
Out of curiosity how long is the feed line into the pump?

**Reply by davthomaspilot** [reply]
It is 6 - 8' from the pump to the injection valve.

But, I embarrassed to say that today I could not reproduce the problem!

Only thing I can think of is that changing the roller assembly  (done a couple of weeks ago) really did fix the problem.  

I concluded it did not, based on the observation ORP wasn't rising above 200-300.  But, there could have been something else going on--especially since I was just getting on board with REM.  

I'm going to repeat the check daily for a while.    I'll remove the tube from the 5 gallon jug while dosing, let the tube fill with air, then insert the tube into a cup of water.  

It really looks like you guys were right!

**Reply by rstrouse** [reply]
Once I thought I was wrong but I was mistaken. :-)
